src/app/_shared/shared.service.ts

```diff
@@ -265,6 +265,11 @@ export class SharedService {
     this.settings.setAll(this.settings.config.time, 'ct-time');
   }
 
+  public getRandomNumber(): number {
+    var min = 0;
+    var max = this.settings.config.message.texts.length;
```

Comment:
Please keep this generic if it is going to be inside of shared.service.ts so it can be used for other things in the future, add the max as a parameter instead.

Reply:
fixed in commit 8925361